com.unity.render-pipelines.universal/Shaders/SimpleLitInput.hlsl

```diff
@@ -41,9 +41,7 @@ half4 SampleSpecularSmoothness(half2 uv, half alpha, half4 specColor, TEXTURE2D_
 #endif
 
 #ifdef _GLOSSINESS_FROM_BASE_ALPHA
-    specularSmoothness.a = exp2(10 * alpha + 1);
```

Comment:
Why was this changed? Is it done somewhere else?

Reply:
Yes - it's moved into the "CalculateBlinnPhong()" function in "Lighting.hlsl", this called from the "UniversalFragmentBlinnPhong()" method which I created to provide a single entry-point for all simple-lit code.